docs/wf_options/Overview.md

```diff
@@ -2,7 +2,7 @@
 
 Workflow options can affect the execution of a single workflow without having to change configuration options or restart Cromwell. 
 
-You provide workflow options to Cromwell in a JSON format. This can be supplied at workflow-submit time either via the [CLI](../CommandLine/) or the [REST endpoint](../api/RESTAPI/):
+You provide workflow options to Cromwell in a JSON format. This can be supplied at workflow-submit time either via the [CLI](../CommandLine.md) or the [REST endpoint](../api/RESTAPI.md):
```

Comment:
Does this render correctly in readthedocs though? I think "local file paths" are not necessarily directly mapped to "hosted URL paths"

Reply:
Tested live, these links do indeed appear to now be broken